tools/autograd/gen_variable_type.py

```diff
@@ -645,14 +620,19 @@ def emit_call(f: NativeFunction, unpacked_bindings: List[Binding]) -> str:
         unpacked_args = [b.name for b in unpacked_bindings]
         base_type_call = emit_dispatch_call(f, 'self_', unpacked_args)
 
+        if get_view_info(fn) is not None or modifies_arguments(f):
+            guard = 'at::AutoNonVariableTypeMode non_var_type_mode(true);'
```

Comment:
nit: I'd aim for consistency in the names, either by renaming this one `guard` or renaming the other one something analogous to `non_var_type_mode` if you don't wanna generate unnecessary codegen diff :D